samples/subsys/usb/dfu-next/README.rst

```diff
@@ -0,0 +1,137 @@
+.. zephyr:code-sample:: dfu-next
+   :name: USB DFU
+   :relevant-api: usbd_api usbd_dfu
+
+   Implement a basic USB DFU device
+
+Overview
+********
+
+This sample application demonstrates the USB DFU implementation using the
+new experimental USB device stack.
+
+Requirements
+************
+
+This project requires an experimental USB device driver (UDC API) and uses the
+:ref:`disk_access_api` and RAM-disk to download/upload the image.
+
+Building and Running
+********************
+
+This sample can be built for multiple boards, in this example we will build it
+for the reel board:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/dfu-next
+   :board: reel_board
```

Comment:
BTW I am assuming you have tested this on the reel_board? :) In general I think it would make more sense that the board here (and other places in the README) would be one of the targets you have as `integration_platforms`
Definitely not worth changing at this point, especially if you're confident this works on reel_board :)

Reply:
This is my favorite board, IMHO the best board ever.

Reply:
Given that @jfischer-no was the original designer of the reel_board I expect he has tested it with it :)

Reply:
Yeah of course, but joke aside I think it's just good practice that we would actually let CI help us keep our docs honest :)